executor/distsql.go

```diff
@@ -655,12 +655,10 @@ func (e *IndexLookUpExecutor) Next(ctx context.Context, req *chunk.Chunk) error
 		if resultTask == nil {
 			return nil
 		}
-		for resultTask.cursor < len(resultTask.rows) {
```

Comment:
What will happen if resultTask.cursor >= len(resultTask.rows)? It will cause an endless loop?

Reply:
i forget  if req.IsFull()  ,so. it is timeout.